eclipse-collections/src/main/java/org/eclipse/collections/impl/list/primitive/IntInterval.java

```diff
@@ -1007,4 +1016,33 @@ public int next()
             throw new NoSuchElementException();
         }
     }
+
+    private class IntIntervalSpliterator implements Spliterator.OfInt {
+        AtomicInteger current = new AtomicInteger(from);
+
+        @Override
+        public OfInt trySplit()
```

Comment:
If `trySplit` is not implemented, there is no splitting, which makes the `SplitIterator` no better than a regular iterator.